rocky/katalogus/views/mixins.py

```diff
@@ -103,7 +103,16 @@ def run_boefje(self, katalogus_boefje: KATalogusBoefje, ooi: Optional[OOI]) -> N
         )
 
         item = QueuePrioritizedItem(id=boefje_task.id, priority=1, data=boefje_task)
-        client.push_task(f"boefje-{self.organization.code}", item)
+        try:
+            client.push_task(f"boefje-{self.organization.code}", item)
+        except Exception as e:
```

Comment:
This exception handling is too broad; why not just catch a `TooManyRequestsError` exception and use a translatable string for the message?